rtf_vis_tool/src/Components/DivGraph.js

```diff
@@ -0,0 +1,164 @@
+                <DivNode textColor={'black'} backgroundColor={lightGray} topOffset={formatPercent(topShift, 78)} leftOffset={formatPercent(leftShift, 10)} text={'Dense Mesh Reconstruction'}/>
+                <DivNode textColor={'white'} backgroundColor={aquaBlue} topOffset={formatPercent(topShift, 83)} leftOffset={formatPercent(leftShift, 1)} text={'Aggregate'}/>
+                <DivNode textColor={'black'} backgroundColor={lightGray} topOffset={formatPercent(topShift, 63)} leftOffset={formatPercent(leftShift, 1)} text={'Zipped Results for All Scenes'}/>
+                <SfMDataDivNode json={da_json} toggleDA_PC={toggleDataAssoc_PointCloud} textColor={'black'} backgroundColor={lightGray} topOffset={formatPercent(topShift, 40)} leftOffset={formatPercent(leftShift, 34)} text={'SfMData'}/>
+                <DivNode textColor={'white'} backgroundColor={aquaBlue} topOffset={formatPercent(topShift, 40)} leftOffset={formatPercent(leftShift, 42)} text={'Data Association w/ Track Filtering'}/>
+                <DivNode textColor={'black'} backgroundColor={lightGray} topOffset={formatPercent(topShift, 53)} leftOffset={formatPercent(leftShift, 45)} text={'Bundler Pinhole Cameras'}/>
+                <DivNode textColor={'white'} backgroundColor={aquaBlue} topOffset={formatPercent(topShift, 53)} leftOffset={formatPercent(leftShift, 53)} text={'Bundler Calibrator'}/>
+                <DivNode textColor={'black'} backgroundColor={lightGray} topOffset={formatPercent(topShift, 43)} leftOffset={formatPercent(leftShift, 60)} text={'absolute Ts'}/>
+                <DivNode textColor={'white'} backgroundColor={aquaBlue} topOffset={formatPercent(topShift, 43)} leftOffset={formatPercent(leftShift, 68)} text={'1d-SfM'}/>
+                <DivNode textColor={'black'} backgroundColor={lightGray} topOffset={formatPercent(topShift, 43)} leftOffset={formatPercent(leftShift, 76)} text={'relative Ts (2): i2ti1'}/>
+                <DivNode textColor={'white'} backgroundColor={aquaBlue} topOffset={formatPercent(topShift, 43)} leftOffset={formatPercent(leftShift, 86)} text={'Largest Connected Component Extractor'}/>
+                <DivNode textColor={'black'} backgroundColor={lightGray} topOffset={formatPercent(topShift, 63)} leftOffset={formatPercent(leftShift, 65)} text={'absolute Rs'}/>
+                <DivNode textColor={'white'} backgroundColor={aquaBlue} topOffset={formatPercent(topShift, 65)} leftOffset={formatPercent(leftShift, 76)} text={'Shonan'}/>
+                <DivNode textColor={'black'} backgroundColor={lightGray} topOffset={formatPercent(topShift, 65)} leftOffset={formatPercent(leftShift, 87)} text={'relative Rs (2): i2Ri1'}/>
+        
+                {/* Render Directed Edges */}
+                {arrowList}
+
+
+                {/* Render Plates */}
+                <div className="scene_optimizer_plate">
+                    <p style={{color: 'red', fontWeight: 'bold'}}>Scene Optimizer Scenes</p>
```

Comment:
plate style in css?

Reply:
you're right. I should add this plate title styling in the css file